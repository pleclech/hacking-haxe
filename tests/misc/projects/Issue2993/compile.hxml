--display Main.hx@76@position