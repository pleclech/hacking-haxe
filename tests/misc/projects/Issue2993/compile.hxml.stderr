<list>
<pos>$$normPath(::cwd::/Main.hx):2: lines 2-5</pos>
</list>
